routes/web.php

```diff
@@ -212,6 +213,18 @@
                 Route::get('/view_answer/{id}', [StudentOperation::class, 'view_answer']);
             });
 
+
+            Route::middleware(['auth:admin'])->group(function () {
```

Comment:
There's already a grouped middleware routes for auth:admin, add these, also we use permission only for access. Give the permission course:update, etc, or create new permissions for this

Reply:
Create new permissions